hedera-node/hedera-mono-service/src/testFixtures/java/com/hedera/test/utils/AddresBookUtils.java

```diff
@@ -41,7 +42,7 @@ public static AddressBook createPretendBookFrom(final Platform platform, final b
         }
         final var nodeId = platform.getSelfId().getIdAsInt();
         final var address = new Address(
-                nodeId,
+                new NodeId(nodeId),
```

Comment:
Same comment as above:

Rather than calling `getIdAsInt` and constructing a new `NodeId` with the int value, you can simply save the `NodeId` returned by `getSelfId`, to pass that into the `Address` constructor

Reply:
updated, thanks!